<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>503</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>723</width>
    <height>503</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>723</width>
    <height>503</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Quick Steg</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="title">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Quick Steg</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Encode Message</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Decode Message</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="message">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>321</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Secret Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>301</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Select File Location:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hide_message">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>460</y>
     <width>131</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Form</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="hidden_message">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>321</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Decoded Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>301</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>File Location:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_hidden_message">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>460</y>
     <width>171</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Show Hidden Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>430</y>
     <width>171</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Form</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>731</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>20</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="write_path">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>140</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="read_path">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>message</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>browse_1</zorder>
  <zorder>hide_message</zorder>
  <zorder>reset_1</zorder>
  <zorder>hidden_message</zorder>
  <zorder>label_6</zorder>
  <zorder>browse_2</zorder>
  <zorder>label_7</zorder>
  <zorder>show_hidden_message</zorder>
  <zorder>reset_2</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>title</zorder>
  <zorder>write_path</zorder>
  <zorder>read_path</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
